lib/task/import/csvImportTask.class.php

```diff
@@ -1038,7 +1072,7 @@ public function execute($arguments = array(), $options = array())
     $import->searchIndexingDisabled = ($options['index']) ? false : true;
 
     // Allow updating to be enabled via a CLI option
-    $import->updateExisting = ($options['update']);
+    $import->updateExisting = isset($options['update']) ? true : false;
```

Comment:
OK last suggestion I promise if it isn't too late! This can just be `isset($options['update'])` since it'll return true/false anyways.